### PR TITLE
Add user in init script.

### PR DESCRIPTION
**Pull request by LeoTindall**

### CODE REVIEW

File: src/server/services/createUser.js [hunk 1]
```diff
@@ -39,15 +39,25 @@ function terminateReadline(message) {
...
+		let emailResult;
+		try {
+			emailResult = await askEmail();
+		} catch (err) {
+			terminateReadline('Invalid email, no user created');
+		}
+		const output = await askPassword(emailResult);
+		email = output[0];
+		password = output[1];
+	} else {
```
Comment (johndiiorio) [inline src/server/services/createUser.js]: This completely skips the check of validating a user's email against the regex.

File: INSTALL.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ You can either use Docker Compose to install Node and PostgreSQL in containers,
 1. Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/).
 1. Clone this repository.
 1. Create a CSV file with a single column called "ip" with your meter IP addresses and copy it into the directory where the project resides.
-1. Set up Node environment and the database by running ```docker-compose run --rm web src/scripts/init.sh <csv file>``` in the main directory. 
+1. Set up Node environment and the database by running ```docker-compose run --rm web src/scripts/init.sh <csv file> --default-user``` in the main directory.
```
Comment (johndiiorio) [inline INSTALL.md]: Recommend against setting up a default user for development users. Perhaps notify them that it is an option, but default to forcing them to make a user.

File: src/scripts/init.sh [hunk 3]
```diff
@@ -62,7 +83,16 @@ if [ $1 != "NONE"  ]; then
     npm run updateMamacMeters 2> /dev/null
 fi
 
+# Create a user
+set -e
+if [ "$DEFAULT_USER" == "yes" ]; then
+    npm run createUser -- test@test.test testtest
+    echo "Created a user 'test@test.test' with password 'testtest'."
+elif [ "$SKIP_USER" != "yes" ]; then 
+    npm run createUser
```
Comment (johndiiorio) [inline src/scripts/init.sh]: Perhaps echo to notify the user that they need to create an admin user

**Comment by LeoTindall** [comment]
@johndiiorio changes addressed

**Pull request merged**